packages/web/components/complex/external-incentivized-pools-table-set.tsx

```diff
@@ -0,0 +1,184 @@
+import { CoinPretty, Dec, PricePretty, RatePretty } from "@keplr-wallet/unit";
+import { ObservablePool } from "@osmosis-labs/stores";
+import { observer } from "mobx-react-lite";
+import { FunctionComponent, useState } from "react";
+import { ExtraGaugeInPool } from "../../config";
+import { useExternalIncentivizedPoolsTable } from "../../hooks/use-external-incentivized-pools-table";
+import { useStore } from "../../stores";
+import { CheckBox, PageList, SortMenu } from "../control";
+import { SearchBox } from "../input";
+import { Table } from "../table";
+import { MetricLoaderCell, PoolCompositionCell } from "../table/cells";
+
+const TVL_FILTER_THRESHOLD = 1000;
+
+export const ExternalIncentivizedPoolsTableSet: FunctionComponent = observer(
+  () => {
+    const {
+      chainStore,
+      queriesExternalStore,
+      priceStore,
+      queriesOsmosisStore,
+      accountStore,
+    } = useStore();
+
+    const chainInfo = chainStore.getChain("osmosis");
+    const queryExternal = queriesExternalStore.get();
+    const queryOsmosis = queriesOsmosisStore.get(chainInfo.chainId);
+    const account = accountStore.getAccount(chainInfo.chainId);
+
+    const [isPoolTvlFiltered, setIsPoolTvlFiltered] = useState(false);
+
+    const externalIncentivizedPools = Object.keys(ExtraGaugeInPool)
+      .map((poolId: string) => {
+        const pool = queryOsmosis.queryGammPools.getPool(poolId);
+        if (pool) {
+          return pool;
+        }
+      })
+      .filter((pool: ObservablePool | undefined): pool is ObservablePool => {
+        if (!pool) {
+          return false;
+        }
+
+        const inner = ExtraGaugeInPool[pool.id];
+        const data = Array.isArray(inner) ? inner : [inner];
```

Comment:
I just migrated this config as well for the pool detail PR. Why isn't every gauge config an array?

Reply:
I think it should change to array

Reply:
I added this config as well for the pool detail branch (to get the bonus guages). I did the same.